app/services/subscriptions/free_trial_billing_service.rb

```diff
@@ -10,36 +10,50 @@ def initialize(timestamp: Time.current)
 
     def call
       ending_trial_subscriptions.each do |subscription|
-        # if subscription.plan_pay_in_advance
-        #   BillSubscriptionJob.perform_later([subscription], timestamp)
-        # end
+        if subscription.plan_pay_in_advance && !subscription.was_already_billed_today &&  !already_billed_on_day_one?(subscription)
```

Comment:
Layout/SpaceAroundOperators: Operator `&&` should be surrounded by a single space. (https://rubystyle.guide#spaces-operators)